frontend/src/shiftpage/shiftpage.tsx

```diff
@@ -34,7 +34,7 @@ const ShiftPage = ({ shiftType }: ShiftPageProps) => {
     const { data: userVolTypesData, isLoading: loadingUserVolTypes } = useVoltypesForUser(userData?.data?._id);
 
     const getFilterInLocalStorage = (): Filters => {
```

Comment:
Maybe it's not a bad idea if the name of the function to be aligned with the functionality 👍